README.md

```diff
@@ -1,2 +1,11 @@
 # git_training
-Git training from the NHS-R community
+Git training from the NHS-R community training steps:
+
+1. Download [R](https://cran.r-project.org/bin/windows/base/) and [RStudio desktop](https://www.rstudio.com/products/rstudio/download/) or create an account on [RStudio cloud](https://login.rstudio.cloud/login?redirect=%2F).
+2. Install Git via this [linnk](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) if you are using RStudio desktop.
```

Comment:
```suggestion
2. Install Git via this [link](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) if you are using RStudio desktop.
```

Reply:
Would add here that if you're using RStudio cloud you do not need this step, right?

Reply:
indeed!